lib/rubocop/cop/rails/file_path.rb

```diff
@@ -34,15 +34,15 @@ class FilePath < Base
         RESTRICT_ON_SEND = %i[join].freeze
 
         def_node_matcher :file_join_nodes?, <<~PATTERN
-          (send (const nil? :File) :join ...)
+          (send (const _ :File) :join ...)
```

Comment:
```suggestion
          (send (const {nil? cbase} :File) :join ...)
```

Reply:
Yes, you are right in pointing this out. You are saying that code like `a::Rails` exists and should be changed like this, right?

Reply:
Yup. It's a false positive.